apps/src/weblab2/CDOIDE/cdoIDEContext/utils.ts

```diff
@@ -27,11 +27,20 @@ import {
 
 const DEFAULT_NEW_FILE_CONTENTS = 'Add your changes to ${fileName}';
 
-export const getNextFileId = (files: ProjectFile[]) =>
-  String(Math.max(...files.map(f => Number(f.id))) + 1);
+export const getNextFileId = (files: ProjectFile[]) => {
+  if (files.length === 0) {
```

Comment:
I discovered if you deleted all the files in the project and tried to create a new one it would have an id of `'-Infinity'` and only one file and one folder could be created. This allows for a default id.